recipe/meta.yaml

```diff
@@ -23,10 +23,10 @@ requirements:
     - setuptools
     - wheel
   run:
-    - python >=3.6
-    - google-api-core >=1.21.0,<3.0.0dev
-    - google-auth >=1.24.0,<3.0.0dev
-    - grpcio >=1.8.2,<2.0.0dev
+    - python
+    - google-api-core >=1.31.6,<3.0.0dev
```

Comment:
Some versions should be excluded:
`- google-api-core >=1.31.6,<3.0.0dev,!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0`
see https://github.com/googleapis/python-cloud-core/blob/57cb3ca14f058c326b15478cd4eacea291020a3e/setup.py#L31

Reply:
thanks for the catch. Done.